tests/test_functions.py

```diff
@@ -10,21 +13,56 @@
 import geoalchemy2.functions  # NOQA
 
 
+SA_VERSION = parse_version(SA_VERSION)
+
+
 def eq_sql(a, b):
     a = re.sub(r'[\n\t]', '', str(a))
     assert a == b
 
 
 def _test_simple_func(name):
-    eq_sql(getattr(func, name)(1).select(),
```

Comment:
I have a last question. I updated these tests for my dev checks but I am not sure it is relevant to change them as they rely on the private object `sqlalchemy.functions._registry`. Should I revert them?

Reply:
Yep, let's avoid relying on SQLAlchemy private stuff. Thanks.